src/PKSim.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -35,11 +39,56 @@ public ProjectMapper(IExecutionContext executionContext, SimulationMapper simula
          snapshot.Formulations = await mapBuildingBlocksToSnapshots<Formulation>(project.All<Model.Formulation>());
          snapshot.Protocols = await mapBuildingBlocksToSnapshots<Protocol>(project.All<Model.Protocol>());
          snapshot.Populations = await mapBuildingBlocksToSnapshots<Population>(project.All<Model.Population>());
-         snapshot.ObservedData = await mapObservedDataToSnapshots(project.AllObservedData);
          snapshot.Simulations = await mapSimulationsToSnapshots(project.All<Model.Simulation>(), project);
+         snapshot.ObservedData = await mapObservedDataToSnapshots(project.AllObservedData);
+         snapshot.ObservedDataClassifications = await mapObservedDataClassificationsToSnapshots(
```

Comment:
if you already have a method `mapObservedDataClassificationsToSnapshots`, why don't you pass it the project directly instead of this long arguments?